packages/jbrowse-plugin-apollo/src/FeatureDetailsWidget/ApolloTranscriptDetailsWidget.tsx

```diff
@@ -112,6 +119,45 @@ export const ApolloTranscriptDetailsWidget = observer(
       props,
     ) as React.ElementType<CustomComponentProps>
 
+    const BiotypesComponent = pluginManager.evaluateExtensionPoint(
+      'Apollo-BiotypesComponent',
+      () => null,
+      {
+        key: 'status',
+        feature,
+        session,
+        assembly,
```

Comment:
I think you can get the assembly from `feature.assemblyId` instead of passing it separately.